CONTRIBUTING.md

```diff
@@ -16,6 +16,9 @@ python setup.py install
 
 # install the developer dependencies (-e is interactive mode)
 pip install -e .[dev]
+
+# MacOS install the developer dependencies (-e is interactive mode)
+pip install -e .'[dev]'  
```

Comment:
I believe the quotes work on both, so perhaps we should just include the example with quotes since it should work for everyone? that is what we do elsewhere. Rather than list this twice.